test/unit/org/apache/cassandra/index/sai/cql/AnalyzerTest.java

```diff
@@ -29,6 +29,19 @@
 
 public class AnalyzerTest extends SAITester
 {
+    @Test
+    public void testEmpty()
```

Comment:
Nit: maybe we could mention compaction in the name, as in `testCompactionWithoutTerms`, or something like that?